content/en/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Kubernetes API health endpoints
+reviewers:
+- logicalhan
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+This page describes the usage of the Kubernetes API endpoints for liveness and readiness.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## API Endpoints for health
+
+Currently the Kubernetes API server supports 3 API endpoints (`healthz`, `livez`, `readyz`) to indicate the current status of the API server.
+Since Kubernetes 1.16 the `healthz` endpoint is deprecated and the more specific `livez` and `readyz` should be used.
+The `livez` endpoint can be used with the `--livez-grace-period` [flag](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver) to specify the startup duration.
+For a graceful shutdown you can specify the `--shutdown-delay-duration` [flag](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver) with the `/readyz` endpoint.
+
+For all endpoints you can use the `verbose` parameter to print out the checks and their status:
+
+```bash
+$ curl -k https://localhost:6443/livez?verbose
+[+]ping ok
```

Comment:
This is not a valid shell script. Please see [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output) for details on the recommended style.